Installation/Action/buren.SyncOpenPDCAction.action.json

```diff
@@ -0,0 +1,19 @@
+{
+    "title": "SyncOpenPDCActionBuren",
```

Comment:
Maybe make this action lockable = true so that it can not run again if a previous run is not done yet